src/client/views/home-view/data-cube-card/data-cube-card.tsx

```diff
@@ -48,8 +48,11 @@ export class DataCubeCard extends React.Component<DataCubeCardProps, DataCubeCar
 
   renderDescription() {
     const { description, extendedDescription } = this.props;
+    const MarkdownNode = React.lazy(() => import(/* webpackChunkName: "markdown-node" */ "../../../components/markdown-node/markdown-node"));
     if (!extendedDescription) {
-      return <MarkdownNode markdown={description || STRINGS.noDescription} />;
+      return <React.Suspense fallback={Loader}>
```

Comment:
Oh my!

Reply:
[It works](https://turnilo-marked-gmbbyye42a-ew.a.run.app/) and how!